element_array_ephys/export/nwb/nwb.py

```diff
@@ -14,12 +14,14 @@
 from spikeinterface import extractors
 from tqdm import tqdm
 import warnings
-from ... import probe, ephys_no_curation
+from ... import probe
+from ... import ephys_no_curation as ephys
 
-assert probe.schema.is_activated(), 'probe not yet activated'
 
-assert ephys_no_curation.schema.is_activated,  \
-        "The ephys module must be activated before export."
+ephys_mode = os.getenv('EPHYS_MODE', dj.config['custom'].get('ephys_mode', 'acute'))
```

Comment:
```suggestion
ephys_mode = os.getenv('EPHYS_MODE', dj.config['custom'].get('ephys_mode'))
```

Reply:
If no value is specified, should we set the default to `None` and then the `NotImplementedError` will also be thrown?

Reply:
This mirrors `acute` as default in the workflow, but either way works in the present case